### ISSUE TITLE
How can i close the debug log?

### ISSUE THREAD

**Issue by T-baby** [stlehmann/Flask-MQTT#42]
How can i close the debug log?

````
DEBUG in __init__: Subscribed to topic: x, qos: 0
````

**Comment by T-baby** [comment]
I also want to be able to implement custom topics:
````python
@mqtt.on_message(topic='')
````

**Comment by T-baby** [comment]
and I can't use the sqlalchemy in on_message:

````python
@mqtt.on_message()
def handle_mqtt_message(client, userdata, message):
    if(message.topic == 'grade-website'):
        start(message.payload.decode())
````

````python
def start(username):
    User.query.filter_by(username=username).first()
````

**Comment by jmrubillon** [comment]
The logging can't be disabled in the current version.
https://github.com/stlehmann/Flask-MQTT/pull/44#issue-233984545
Addresses this.

**Comment by stlehmann** [comment]
This should be closed with 093e4a09416c41c592915cbf90f166d5e3141995. Thanks @jmrubillon for the PR.

**Comment by jmrubillon** [comment]
You're welcome. I'd forgotten I even contributed those PRs :)
